package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Book library management system",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha ./server/test --require @babel/register --colors --slow 400 --timeout 5000 --exit",
+    "test": "npm run clean-tables && cross-env NODE_ENV=test nyc --reporter=text mocha ./server/test/*.spec.js --require @babel/register --require @babel/polyfill --slow 400 --timeout 5000 --exit",
```

Comment:
Bro, well done for adding the `babel polyfill` plugin to the test script. However, if I may ask, why did you remove the `--colors` option?

Reply:
Thanks. I  might have missed when resolving conflicts. I will add it in my next commit.

Reply:
Thanks. I  might have missed when resolving conflicts. I will add it in my next commit.